frontend/src/App.tsx

```diff
@@ -1,35 +1,111 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useRef, useState } from "react";
+import reactLogo from "./assets/react.svg";
+import viteLogo from "/vite.svg";
+import "./App.css";
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [count, setCount] = useState(0);
+  const [callButtonActive, setCallButtonActive] = useState(false);
+  const [hangupButtonActive, setHangupButtonActive] = useState(false);
+  const [startButtonActive, setStartButtonActive] = useState(true);
+  const [status, setStatus] = useState("연결 대기중");
+
+  const localVideoRef = useRef<HTMLVideoElement | null>(null);
+  const remoteVideoRef = useRef<HTMLVideoElement | null>(null);
+  // WebRTC 설정
+  const configuration = {
+    iceServers: [
+      {
+        urls: "stun:stun.l.google.com:19302",
+      },
+    ],
+  };
```

Comment:
STUN 서버를 저희 서버로 해주시면 좋을 것 같네요

Dev 서버 일 경우 해당 서버를 사용하시고, 실제 production 빌드에서는 저희 서버로 쓰도록 만들어주시면 좋을 것 같습니다~
```suggestion
    iceServers: [
      {
        urls: "stun:stun.l.google.com:19302",
      },
    ],
  };
```